beacon-chain/sync/service.go

```diff
@@ -296,7 +295,7 @@ func (s *Service) Status() error {
 // and prevent DoS.
 func (s *Service) initCaches() {
 	s.seenBlockCache = lruwrpr.New(seenBlockSize)
-	s.seenBlobCache = lruwrpr.New(seenBlobSize)
+	s.seenBlobCache = lruwrpr.New(seenBlockSize * params.BeaconConfig().MaxBlobsPerBlock(s.cfg.clock.CurrentSlot()))
```

Comment:
To make this BPO friendly, could we set it to the max of the sizes seen in the bpo schedule?

Reply:
When I say "BPO friend", what I meant is keeping in mind that BPOs will raise the max without a restart. So we should take that into account and pick a value that will be appropriate as we go through the schedule. IMO that points to using the highest value found in the schedule.

Reply:
Oh yeah, I think I mentioned it in the description, this PR fixes the bug where a node restart wouldn’t pick up the correct value. I could make it more dynamic so it resizes at every new BPO window, but that approach wouldn't be as simple. Do you want me to go that route instead?

Reply:
What about just using:
```go
params.BeaconConfig().MaxBlobsPerBlock(math.MaxUint64)
```
?

I agree it's not optimal, but actually I think we don't care, since, sooner or later, we'll get this BPO value.
IMO reaching optimality of cache size does not worth the complexity.